kafka-daq-report/Cargo.toml

```diff
@@ -1,23 +1,23 @@
 [package]
 name = "kafka-daq-report"
-version = "0.1.0"
-license = "GPL-3.0-only"
-edition = "2021"
+version.workspace = true
+license.workspace = true
+edition.workspace = true
 
 [dependencies]
 common = { path = "../common" }
 streaming-types = { path = "../streaming-types" }
 
-anyhow = "1.0"
-chrono = "0.4.22"
-clap = { version = "4.0", features = ["derive"] }
+anyhow.workspace = true
+chrono.workspace = true
+clap.workspace = true
 crossterm = "0.26.1"
-env_logger = "0.10"
+env_logger.workspace = true
 # hdf5 = { version = "0.8.1", features = ["static"] }
```

Comment:
Keep this with the workspace level `hdf5` (it needs to be reverted back to this after the next release of this crate).

Reply:
The commented out line should be moved to the workspace level `Cargo.toml` file, and removed from the `Cargo.toml` files in each crate.